tools/test-proxy/documentation/asset-sync/README.md

```diff
@@ -75,13 +77,115 @@ Reset discards local changes to a targeted assets.json files and resets the loca
+| +----5hgHKwvMaN/       |   |  |
+|        <sparse clone>  |   |  |
+|      AuN9me8zrT--------+   |  |
+|        <sparse clone>      |  |
+|      BSdGcyN2XL------------+  |
+|        <sparse clone>         |
++-------------------------------+
+```
+
+As you run tests in `recording` or `playback` mode, the `test-proxy` automatically checks out the appropriate tag in each local assets repo. After running docs in `record` mode, the newly updated recordings will be sitting within the appropriate assets repository.
+
+To view the changes before pushing, use one of the one-liners defined below [below.](#attempt-to-manually-resolve).
+
+### My tests don't use the test-proxy at all currently, how do I externalize my recordings?
+
+You don't. Your first step is to integrate your test framework with the `test-proxy`.
+
+Refer to:
+
+- [Test-Proxy Readme](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md)
+- [Test-Proxy Sample Recording Clients](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy/sample-clients)
+
+### I'm a dev who uses the test-proxy currently, how do I externalize my recordings?
+
+First, ensure that your language-specific "shim" supports the automatic addition of the `x-recording-assets-file` key to the test-proxy `Record|Playback/Start/` endpoints.
+
+- [Enabling in .NET](https://github.com/Azure/azure-sdk-for-net/pull/31157)
+- [Enabling in Python](https://github.com/Azure/azure-sdk-for-python/pull/26078)
+- [Enabling in JS](https://github.com/Azure/azure-sdk-for-js/pull/23405)
+- [Enabling in Go(https://github.com/Azure/azure-sdk-for-go/pull/19322)
+
+Use [the transition script](../../scripts/transition-scripts/generate-assets-json.ps1) and follow the [readme](../../scripts/transition-scripts/README.md)!
+
+### What does this look like in practice?
+
+TODO: complete "run in record mode, look at recordings, test-proxy push" section
+
+### I am getting weird errors out of my test-proxy operations
+
+If you think that the `test-proxy` has somehow gotten itself into a weird "in-between" state that it can't automatically dig itself out of, you have a couple options.
+
+#### Reset it
+
+The first, most foolproof, and most destructive of options.
+
+```powershell
+test-proxy reset --assets-json-path <assetsJsonPath>
+```
+
+This will _force_ the locally cloned assets to align with the assets.json that has been targeted.
+
+#### Attempt to manually resolve
+
+A **new tag** is pushed with each `test-proxy push` invocation. There should be _no such thing_ as `merge conflicts` when automatically pushing up a new tag. However, if you wish to manually resolve instead of discarding current state, continue reading this section.
+
+The below diagram illustrates how your assets.json, language repo, and assets repo relate to each other. Text that appears as a specific color can be traced to its source in the assets.json.
+
+![assets diagram](../_images/organization_of_assets.png)
```

Comment:
I am very confused as to why this is showing up as broken.

Reply:
Based on the fact that the asset sync block diagram is also showing up as broken in this PR, I wonder if it's a PR preview issue more than an actual issue